packages/rancher-pushprox/Chart.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+appVersion: 1.16.0
+annotations:
```

Comment:
Unclear if these are needed at this level, but let's leave them in for now.

Reply:
I updated the appVersion to 0.1.0 to match the version of the upstream PushProx we are deploying:

https://github.com/rancher/PushProx/blob/753361c573410dc535028be56022962b9df3b9a6/VERSION#L1